checker/src/org/checkerframework/checker/upperbound/UpperBoundAnnotatedTypeFactory.java

```diff
@@ -151,7 +167,7 @@ public Integer valMaxFromExpressionTree(ExpressionTree tree) {
      *
      * @return annotation given by name with names=values, or UNKNOWN
      */
-    private static AnnotationMirror createAnnotation(String name, String[] names) {
+    static AnnotationMirror createAnnotation(String name, String[] names) {
```

Comment:
I can make it explicitly protected, i need it to be available to the Store so that i can make annotations for the remove and clear stuff.